app/src/views/private/components/drawer-item.vue

```diff
@@ -289,13 +291,13 @@ function useItem() {
 	return { internalEdits, loading, initialValues, fetchItem };
 
 	async function fetchItem() {
-		loading.value = true;
-
 		if (!props.primaryKey) return;
 
+		loading.value = true;
```

Comment:
this is another unrelated change, but currently `loading.value` is set to true before the early return statement, so if it ever does get returned early, the loading will never stop. Opted to move it below the return statement to prevent this edge case from happening.